app/_gateway_entities/route.md

```diff
@@ -45,31 +51,36 @@ Routes fulfill two responsibilities in {{ site.base_gateway }}:
 1. Match incoming requests and route them to the correct [Gateway Service](/gateway/entities/service/)
 2. Use plugins to transform the request/response proxied using this Route
 
-A Route must be attached to a [Service](/gateway/entities/service/), and _may_ have one or more [Plugin](/gateway/entities/plugin/) entities attached.
+A Route must be attached to a [Gateway Service](/gateway/entities/service/), and _may_ have one or more [Plugin](/gateway/entities/plugin/) entities attached.
 
-## Route and Service interaction
+## Route and Gateway Service interaction
 
-Routes, in conjunction with [Services](/gateway/entities/service/), let you expose your services to applications with {{site.base_gateway}}. Routes also allow the same service to be used by multiple applications and apply different policies based on the route used.
+Routes, in conjunction with [Gateway Services](/gateway/entities/service/), let you expose service applications to clients with {{site.base_gateway}}. 
```

Comment:
```suggestion
Routes, in conjunction with [Gateway Services](/gateway/entities/service/), let you expose service applications to clients with {{site.base_gateway}}. A service application is an application you own that serves requests and responses.
```

Reply:
IMHO this definition shouldn't be necessary